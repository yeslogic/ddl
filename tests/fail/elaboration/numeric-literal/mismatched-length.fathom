//~ exit-code = 1

[3, 4] : Array64 12 U32
