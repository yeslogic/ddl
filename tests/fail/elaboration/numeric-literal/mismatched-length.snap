stdout = ''
stderr = '''
error: mismatched array length
  ┌─ tests/fail/elaboration/numeric-literal/mismatched-length.fathom:3:1
  │
3 │ [3, 4] : Array64 12 U32
  │ ^^^^^^ array with invalid length
  │
  = expected length 12
  =    found length 2

'''
